resources/lang/en/contest.php

```diff
@@ -51,6 +52,8 @@
     'vote' => [
         'list' => 'votes',
         'count' => '1 vote|:count votes',
+        'points' => 'points',
+        'votes' => 'votes',
```

Comment:
there's already `count` above; the `1 vote` should also probably be replaced with `:count vote` to minimise issues with plurality rules